sys-kernel/genkernel/genkernel-9999.ebuild

```diff
@@ -164,72 +163,20 @@ src_prepare() {
 	fi
 
 	# Update software.sh
-	sed -i \
-		-e "s:VERSION_BCACHE_TOOLS:${VERSION_BCACHE_TOOLS}:"\
-		-e "s:VERSION_BOOST:${VERSION_BOOST}:"\
-		-e "s:VERSION_BTRFS_PROGS:${VERSION_BTRFS_PROGS}:"\
-		-e "s:VERSION_BUSYBOX:${VERSION_BUSYBOX}:"\
-		-e "s:VERSION_COREUTILS:${VERSION_COREUTILS}:"\
-		-e "s:VERSION_CRYPTSETUP:${VERSION_CRYPTSETUP}:"\
-		-e "s:VERSION_DMRAID:${VERSION_DMRAID}:"\
-		-e "s:VERSION_DROPBEAR:${VERSION_DROPBEAR}:"\
-		-e "s:VERSION_EUDEV:${VERSION_EUDEV}:"\
-		-e "s:VERSION_EXPAT:${VERSION_EXPAT}:"\
-		-e "s:VERSION_E2FSPROGS:${VERSION_E2FSPROGS}:"\
-		-e "s:VERSION_FUSE:${VERSION_FUSE}:"\
-		-e "s:VERSION_GPG:${VERSION_GPG}:"\
-		-e "s:VERSION_HWIDS:${VERSION_HWIDS}:"\
-		-e "s:VERSION_ISCSI:${VERSION_ISCSI}:"\
-		-e "s:VERSION_JSON_C:${VERSION_JSON_C}:"\
-		-e "s:VERSION_KMOD:${VERSION_KMOD}:"\
-		-e "s:VERSION_LIBAIO:${VERSION_LIBAIO}:"\
-		-e "s:VERSION_LIBGCRYPT:${VERSION_LIBGCRYPT}:"\
-		-e "s:VERSION_LIBGPGERROR:${VERSION_LIBGPGERROR}:"\
-		-e "s:VERSION_LIBXCRYPT:${VERSION_LIBXCRYPT}:"\
-		-e "s:VERSION_LVM:${VERSION_LVM}:"\
-		-e "s:VERSION_LZO:${VERSION_LZO}:"\
-		-e "s:VERSION_MDADM:${VERSION_MDADM}:"\
-		-e "s:VERSION_MULTIPATH_TOOLS:${VERSION_MULTIPATH_TOOLS}:"\
-		-e "s:VERSION_POPT:${VERSION_POPT}:"\
-		-e "s:VERSION_STRACE:${VERSION_STRACE}:"\
-		-e "s:VERSION_THIN_PROVISIONING_TOOLS:${VERSION_THIN_PROVISIONING_TOOLS}:"\
-		-e "s:VERSION_UNIONFS_FUSE:${VERSION_UNIONFS_FUSE}:"\
-		-e "s:VERSION_USERSPACE_RCU:${VERSION_USERSPACE_RCU}:"\
-		-e "s:VERSION_UTIL_LINUX:${VERSION_UTIL_LINUX}:"\
-		-e "s:VERSION_XFSPROGS:${VERSION_XFSPROGS}:"\
-		-e "s:VERSION_XZ:${VERSION_XZ}:"\
-		-e "s:VERSION_ZLIB:${VERSION_ZLIB}:"\
-		-e "s:VERSION_ZSTD:${VERSION_ZSTD}:"\
-		"${S}"/defaults/software.sh \
-		|| die "Could not adjust versions"
+	export VERSION_BCACHE_TOOLS VERSION_BOOST VERSION_BTRFS_PROGS VERSION_BUSYBOX VERSION_COREUTILS
+	export VERSION_CRYPTSETUP VERSION_DMRAID VERSION_DROPBEAR VERSION_EUDEV VERSION_EXPAT VERSION_E2FSPROGS
+	export VERSION_FUSE VERSION_GPG VERSION_HWIDS VERSION_ISCSI VERSION_JSON_C VERSION_KMOD VERSION_LIBAIO
+	export VERSION_LIBGCRYPT VERSION_LIBGPGERROR VERSION_LIBXCRYPT VERSION_LVM VERSION_LZO VERSION_MDADM
+	export VERSION_POPT VERSION_STRACE VERSION_THIN_PROVISIONING_TOOLS VERSION_UNIONFS_FUSE VERSION_USERSPACE_RCU
+	export VERSION_UTIL_LINUX VERSION_XFSPROGS VERSION_XZ VERSION_ZLIB VERSION_ZSTD VERSION_KEYUTILS
 }
 
 src_compile() {
-	if [[ ${PV} == 9999* ]] ; then
-		emake
-	fi
+	emake PREFIX=/usr
 }
 
 src_install() {
-	insinto /etc
-	doins "${S}"/genkernel.conf
-
-	doman genkernel.8
```

Comment:
subtle bug introduced here. If /usr/share/man8 does not exist, the codebase creates /usr/share/man/man8.bz2 with that content.

Reply:
I fixed it in the main repo